x-pack/plugins/cloud/public/plugin.ts

```diff
@@ -25,6 +25,7 @@ export interface CloudConfigType {
 
 interface CloudSetupDependencies {
   home?: HomePublicPluginSetup;
+  security?: Pick<SecurityPluginSetup, 'authc'>;
```

Comment:
NIT: as you're using the whole type in the tests via `securityMock` anyhow, any reason to use a `Pick` here?

Reply:
I leave that to Larry, but I personally like the fact that we explicitly outline the part of the dependency contract we really need (too bad we don't have a more expressive syntax to do that). This can make the life much easier if you're changing/fixing the behavior of a certain portion of your public contract and want to quickly know what plugins may be affected without following all the paths where the contract is passed to.

Reply:
Yeah I did this mostly out of habit for the reasons Aleh mentioned. I don't mind reverting if you'd prefer the original approach

Reply:
Na, this is fine